src/components/core/mocks.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
```

Comment:
This import is needed when the JSX is used in the file.
There's no JSX in this file - therefore no React import is needed.

Reply:
done